dev-python/pytest-xvfb/pytest-xvfb-2.0.0.ebuild

```diff
@@ -21,9 +21,8 @@ DEPEND="
 	x11-base/xorg-server[xvfb]
 "
 
-distutils_enable_tests pytest
+# Needs to be installed to system for tests to work at all
+# distutils_install_for_testing does not work
+BDEPEND="test? ( dev-python/pytest-xvfb[${PYTHON_USEDEP}] )"
```

Comment:
This looks wrong to me. Please have a look at https://dev.gentoo.org/~mgorny/python-guide/distutils.html#installing-the-package-before-testing

Reply:
>  Please have a look at https://dev.gentoo.org/~mgorny/python-guide/distutils.html#installing-the-package-before-testing
I tried that (in fact that is how it was before) and it didn't work, this is not the first pytest-foobar package where I found that the `distutils_install_for_testing` function is not sufficient to get the tests working. The only solution I can think of is to have the tests of the package BDEPEND on itself, though it is a very ugly solution IMO.

Reply:
Are you sure that was the reason tests were failing? They work fine here and the original bug doesn't seem to have much to do with the plugin not being found.
I'd say it looks more like a X-related failure. I'm attempting to reproduce on a cleaner system since it works on my desktop.

Reply:
> Are you sure that was the reason tests were failing?
Yes I could reproduce the error by removing the package and trying to reinstall it.
emerge -C pytest-xvfb && emerge -1 pytest-xvfb fails
USE="-test" emerge -1 pytest-xvfb && emerge -1 pytest-xvfb works

On the other hand, the fact that it is the way it is in the ebuild, implies that I got it working before with the `distutils_install_for_testing` function, though it is too long ago for me to remember.

Reply:
Can you paste the failure with the current ebuild (pre-PR)? I'm curious.

Reply:
OK I can see the failure on another system when the package isn't installed. Installing it makes it work. That's really weird.